src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -161,6 +161,7 @@ def __init__(self):
         self.qr_rules = {}
         # Builtins cannot request mesh properties
         self.mesh = None
+        self._idx_name = None
```

Comment:
What's the reason for this addition?

Reply:
This fixes a pylint warning (attribute defined outside of constructor or so).

Reply:
Oh yes.